src/sardana/taurus/qt/qtgui/extra_sardana/measurementgroup.py

```diff
@@ -125,6 +125,8 @@ def createChannelDict(channel, index=None, **kwargs):
         #           'timer': '', #should contain a channel name
         #           'monitor': '', #should contain a channel name
         #           'trigger': '', #should contain a channel name
+        'value_ref_enabled': True,  # bool
```

Comment:
For the moment It's better to use default value false. we won't distinguish between the exp channel types neither itf these are referable or not, so it does not have too much sense to leave it empty. Just changing the default value is enough. Thanks @mrosanes!